src/build/transforms/transform-pvpoke-move.ts

```diff
@@ -0,0 +1,38 @@
+import { 
+  BuffedChargeMove, FastMove, ChargeMove, BuffTarget 
+} from '../entities/moves/types'
+
+export const transformPvpokeMove = ( m: any ) => {
+  const {
```

Comment:
Consider validating the incoming data so that we get a useful error message next time pvpoke changes their data structure.